src/main/java/pl/futurecollars/invoicing/config/Config.java

```diff
@@ -1,8 +1,9 @@
 package pl.futurecollars.invoicing.config;
 
+@SuppressWarnings("unused")
 public class Config {
```

Comment:
Skoro ta klasa nie jest nam już potrzebna to na spokojnie możemy ją usunąć. Dodawanie @SuppressWarnings nie jest dobrą praktyką.